### ISSUE TITLE
JSON Render error when executing salt-ssh

### ISSUE THREAD

**Issue by rafalwrzeszcz** [saltstack/salt#54990]
### Description of Issue
Executing `salt-ssh` fails with an error:

```
$ sudo salt-ssh --state-verbose=True --state-output full -v -i --priv /home/rafal.wrzeszcz/.ssh/id_rsa test0 state.apply test=True
Executing job with jid 20191013192526721187
-------------------------------------------


[ERROR   ] JSON Render failed for: 
Traceback (most recent call last):
  File "/var/tmp/.admin_d7d956_salt/salt-call", line 27, in <module>
    salt_call()
  File "/var/tmp/.admin_d7d956_salt/pyall/salt/scripts.py", line 431, in salt_call
    client.run()
  File "/var/tmp/.admin_d7d956_salt/pyall/salt/cli/call.py", line 57, in run
    caller.run()
  File "/var/tmp/.admin_d7d956_salt/pyall/salt/cli/caller.py", line 138, in run
    ret = self.call()
  File "/var/tmp/.admin_d7d956_salt/pyall/salt/cli/caller.py", line 237, in call
    ret['return'] = self.minion.executors[fname](self.opts, data, func, args, kwargs)
  File "/var/tmp/.admin_d7d956_salt/pyall/salt/executors/direct_call.py", line 12, in execute
    return func(*args, **kwargs)
  File "/var/tmp/.admin_d7d956_salt/pyall/salt/modules/state.py", line 2198, in pkg
    _set_retcode(ret)
  File "/var/tmp/.admin_d7d956_salt/pyall/salt/modules/state.py", line 107, in _set_retcode
    if not __utils__['state.check_result'](ret, highstate=highstate):
  File "/var/tmp/.admin_d7d956_salt/pyall/salt/loader.py", line 1208, in __getitem__
    func = super(LazyLoader, self).__getitem__(item)
  File "/var/tmp/.admin_d7d956_salt/pyall/salt/utils/lazy.py", line 105, in __getitem__
    return self._dict[key]
KeyError: 'state.check_result'
[ERROR   ] Expecting value: line 1 column 1 (char 0)
test0:
```

### Setup
Salt managed to execute simple states, like `mc: pkg.installed` but fails on any `service.*` or `augeas.*` (not tried anything further).

### Steps to Reproduce Issue
```
openssh-server:
    pkg.installed

python3-augeas:
    pkg.installed

/etc/ssh/sshd_config:
    augeas.change:
        - require:
            - pkg: "openssh-server"
            - pkg: "python3-augeas"
        - context: "/files/etc/ssh/sshd_config"
        - changes:
            - "set PermitRootLogin no"
            - "set PasswordAuthentication no"
```
(Include debug logs if possible and relevant.)

### Versions Report
```
$ salt-ssh --versions-report
Salt Version:
           Salt: 2019.2.1
 
Dependency Versions:
           cffi: Not Installed
       cherrypy: Not Installed
       dateutil: 2.7.3
      docker-py: Not Installed
          gitdb: Not Installed
      gitpython: Not Installed
          ioflo: Not Installed
         Jinja2: 2.10
        libgit2: Not Installed
        libnacl: Not Installed
       M2Crypto: Not Installed
           Mako: Not Installed
   msgpack-pure: Not Installed
 msgpack-python: 0.5.6
   mysql-python: Not Installed
      pycparser: Not Installed
       pycrypto: 2.6.1
   pycryptodome: Not Installed
         pygit2: Not Installed
         Python: 3.7.3 (default, Apr  3 2019, 05:39:12)
   python-gnupg: Not Installed
         PyYAML: 3.13
          PyZMQ: Not Installed
           RAET: Not Installed
          smmap: Not Installed
        timelib: Not Installed
        Tornado: 4.5.3
            ZMQ: Not Installed
 
System Versions:
           dist: debian 10.1 
         locale: UTF-8
        machine: x86_64
        release: 4.9.0-3-amd64
         system: Linux
        version: debian 10.1
```

(on the target system there is clean Debian AWS AMI)

**Comment by garethgreenaway** [comment]
@rafalwrzeszcz Thanks for the report.  I was unable to reproduce this using the latest 2019.2.1 branch and I believe this should be fixed in the upcoming 2019.2.2 release which is scheduled to be available in the next couple weeks.

**Comment by rafalwrzeszcz** [comment]
I dug a little deeper and found out it was failing on states autoloading. Particularly it was trying to load `pip` modules. Adding `python3-pip` to my schema with `order: 1` fixed the problem. Maybe it's worth adding some checks to print meaningful error message?

**Comment by einsiedlerkrebs** [comment]
I'am experiencing the same issue on more complex states using pyobjects renderer. with and without the `use_yamlloader_old: True` option set for salt-ssh minions.

**Comment by einsiedlerkrebs** [comment]
This issue seem to be fixed with the 3000.1 release!
